### DISCUSSION TITLE
SO 2.3.50 now available!

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#4021] [Announcements]
https://blog.securityonion.net/2021/04/security-onion-2350-now-available.html

![Screen Shot 2021-04-28 at 8 43 47 AM](https://user-images.githubusercontent.com/1659467/116421373-c4d06e00-a80c-11eb-8976-4f220a2b5f60.png)

**Comment by wtl06639** [comment]
When trying to update i get the following error:

Command failed with exit code 1; will retry in 10 seconds (1 / 50)...
Executing command with retry support: docker pull ghcr.io/security-onion-solutions/so-domainstats:2.3.50
Error response from daemon: name unknown
Results:  (1)

which continues until it fails for good

**Reply by TOoSmOotH** [reply]
Please start your own thread
